book/src/offline-signing/README.md

```diff
@@ -0,0 +1,78 @@
+# Offline Transaction Signing
+
+Some security models require keeping signing keys, and thus the signing
+process, separated from transaction creation and network broadcast. Examples
+include:
+  * Collecting signatures from geographically disparate signers in a
+multi-signature scheme
+  * Signing transactions using an [airgapped](https://en.wikipedia.org/wiki/Air_gap_(networking))
+signing device
+
+To facilitate these operations, it is necessary to allow transaction
+construction and signing to occur independently.
+
+This document describes using Solana's CLI tools achieve these goals.
```

Comment:
The first sentence sure uses a lot of long words, and I'm not sure you need it at all. How about just something like:
"This document describes using the Solana CLI to sign and then separately construct/submit a transaction."